### ISSUE TITLE
modprobe: FATAL: Module darling-mach not found in directory /lib/modules/5.4.0-39-generic

### ISSUE THREAD

**Issue by hongyi-zhao** [darlinghq/darling#840]
Hi,

With the latest release of darlinghq installed on Ubuntu 20.04, I meet the following problem:

```
werner@X10DAi-01:~$ darling shell
modprobe: FATAL: Module darling-mach not found in directory /lib/modules/5.4.0-39-generic
Failed to load the kernel module
```

How to solve it?

**Comment by LubosD** [comment]
You don't have the kernel module installed for the running kernel.

**Comment by hongyi-zhao** [comment]
Thanks. In fact, I installed the following package. 

`$ sudo gdebi darling-dkms_0.1.20200331.testing_amd64.deb`

But the problem is caused by the kernel version for which I install this package. In detail, the Ubuntu 20.04 has several kernel version exist under /boot directory and I can boot the OS with grub2 or refind. OTOH, the grub2 and refind use different kernel versions. Normally, I boot the OS with refind which use the latest kernel version under /boot directory, but occasionally, I booted the OS via grub2 and installed the darling-dkms package. After I reboot my OS, I select to use the refind as the bootloader, so the problem happened.

Now, I reinstalled the darling-dkms for the currently active kernel and the problem solved. 

Thanks again.
